data/metadata.yaml

```diff
@@ -3,6 +3,7 @@ linkcolor: blue
 urlcolor: blue
 toccolor: black
 
+normcolor: 888888
```

Comment:
Let's go with `uccolor` here for unchanged color. I feel like the "normal" thing makes sense in the context of the generic color scheme, but these colors are `diff` specific.

Reply:
Feels a bit silly asking this, but can we go with `898887` here?
It's the grey code from the `kate` highlighting color scheme.

Reply:
Would be even sillier if I said no.